src/components/ItemList/ItemList.tsx

```diff
@@ -0,0 +1,178 @@
+import { useState, useEffect, useCallback, createContext } from 'react';
+import { NavLink, Outlet, useNavigate, useParams } from 'react-router-dom';
+import classes from './ItemList.module.css';
+import Paginator from '../Paginator/Paginator';
+import Loader from '../Loader/Loader';
+import { queryItems } from '../API/API';
+import { EMPTY_ITEMS_ARRAY } from './data';
+import { SEARCH_DEFAULT } from '../../data';
+import { MAX_AMOUNT } from '../../data';
+
+export const DataFromChildContext = createContext<React.Dispatch<
+  React.SetStateAction<boolean>
+> | null>(null);
+
+const ItemList = function () {
+  const [beerList, setBeerList] = useState(EMPTY_ITEMS_ARRAY);
+  const [isLoading, setIsLoading] = useState(false);
+  const [requestOK, setRequestOK] = useState(true);
+  const [sectionOpen, setSectionOpen] = useState(false);
+  const [searchText, setSearchText] = useState('');
+  const [pageNumber, setPageNumber] = useState(1);
+  const [perPage, setPerPage] = useState(4);

```

Comment:
It will be good to add variable for this value

Reply:
Fixed in https://github.com/ylakhtin/rs-react-components/commit/fb873d74a06f67fee2d0e4ab1162cf011fe4b438